CVChatbot/CVChatbot.Bot/ChatbotActions/ChatbotAction.cs

```diff
@@ -50,15 +50,14 @@ public bool DoesChatMessageActiveAction(Message incomingMessage, bool isMessageA
         protected abstract string GetRegexMatchingPattern();
 
         /// <summary>
-        /// Returns the regex object needed for pattern matching with the incoming chat message.
         /// This is already populated with the necessary matching pattern text.
         /// You may call this method from the RunAction() method if you need arguments within chat message.
         /// </summary>
-        /// <returns></returns>
+        /// <returns>The regex object needed for pattern matching with the incoming chat message.</returns>
         protected Regex GetRegexMatchingObject()
         {
             var pattern = GetRegexMatchingPattern();
-            return new Regex(pattern, RegexOptions.IgnoreCase);
+            return new Regex(pattern, RegexOptions.IgnoreCase | RegexOptions.CultureInvariant);
```

Comment:
Ha, I actually thought about using that flag. But in this case I'd need to run some benchmarks first. Since using the `Complied` flag allows for better matching performance, but, at a cost of greater initialisation time.